### PR TITLE
Avoid Ansible shell module

### PR DESCRIPTION
**Pull request by Duologic**
I've 'ansible-ified' this a bit. Didn't test it thoroughly tho.

### CODE REVIEW

**Comment by markscarbrough** [comment]
I second this pull request.

**Comment by Duologic** [comment]
Rebased on master.

**Comment by capotej** [comment]
Looks great, thanks! :+1:

File: tasks/main.yml [hunk 1]
```diff
@@ -65,29 +69,51 @@
...
-  register: added_rpm_repository
+  get_url:
+      url: "{{ redhat_config_file_url }}"
+      dest: "{{ redhat_config_file_location }}"
+      url_username: "{{ master_token }}"
+      force_basic_auth: yes
+  register: added_rpm_repository_with_token
   when: ansible_os_family == "RedHat" and master_token is defined
 
+- name: Disable repo_gpgcheck !workaround!
```
Comment (capotej) [inline tasks/main.yml:103]: Can you explain why this workaround was added?

Comment (capotej) [reply tasks/main.yml:103]: Saw the [commit message associated with this](https://github.com/computology/packagecloud-ansible-role/pull/4/commits/444080efe99bd40001866c8e5b04ddb2dbe7542e), that issue is not related to ansible at all so I removed it. That error was likely related to our recent GPG key migration https://blog.packagecloud.io/eng/2018/10/17/gpg-key-migration/

Comment (Duologic) [reply tasks/main.yml:103]: I can confirm my playbooks work without this workarond. The commit was from over a year ago, so I assume Packagecloud fixed it by now.

**Comment by Duologic** [comment]
Thanks for the merge.

Can you also import it into the Ansible Galaxy? It looks like that it didn't happen for a long time and now it doesn't show up in 'ansible-galaxy search packagecloud' nor does the install work.

**Comment by capotej** [comment]
Is there anything I can do beyond what is here?: https://galaxy.ansible.com/computology/packagecloud

**Comment by Duologic** [comment]
You can click the import button again after the code changes on your account. (on https://galaxy.ansible.com/my-content/namespaces)

This can be automated from the CI tooling: https://github.com/Duologic/ansible-role-sentry/blob/master/.travis.yml#L28

**Pull request merged**